joomla/joomla-cms
# [4.0] Fix installer file input width + other tweaks

Pull Request for Issue #14657

### Summary of Changes

This PR:

- Fixes the installer file type input width issue so text doesn't overflow
- Replace a couple of instances of jQuery with vanilla JS
- Some styling tweaks
- Replace `<input type="button">` with a normal `<button>`
- Fixes the alert cog not directing the user to the options page.


### Testing Instructions

1. [**com_installer**] Ensure text doesn't overflow when using long file names
2. [**com_installer**] Go to the Folder tab, type in `http://google.com`, click `Install` and ensure the "loading" overlay still appears
3. [**com_installer**] Click the cog icon in the alert. This should now direct you to the correct page rather than just close the alert.
4. [**com_joomlaupdate**] Ensure text doesn't overflow when using long file names
